core/payment/src/models/invoice_event.rs

```diff
@@ -50,7 +50,7 @@ impl WriteObj {
             owner_id,
             event_type: event_type.to_string(),
             details,
-            timestamp,
+            timestamp: Utc::now().adapt(),
```

Comment:
It seems like whole block from lines 34 to 47 is unnecessary now. I wonder why warning is not emitted? :fearful:

Reply:
Thanks for noticing. I was convinced, I removed this.